app/templates/macros/toolkit_forms.html

```diff
@@ -2,7 +2,9 @@
   {%
     with
     question=question_content.question|markdown,
-    hint=(question_content.hint or '')|markdown,
+    question_advice=question_content.question_advice,
+    hint=question_content.hint,
```

Comment:
Are we sure about losing the `|markdown` from `hint`?
I'm pretty sure that some of the hints have markdown-style links in them. Perhaps only on the supplier-app questions, I'm not sure, but even so it might be nice to have consistency of treatment of the content keys across apps - we don't want to have to remember that if we're doing hints that are rendered by the supplier app we can use markdown but if in the buyer app we can't.